trino/dbapi.py

```diff
@@ -401,7 +401,11 @@ def _deallocate_prepare_statement(self, added_prepare_header, statement_name):
     def _generate_unique_statement_name(self):
         return 'st_' + uuid.uuid4().hex.replace('-', '')
 
-    def execute(self, operation, params=None):
+    def execute(self, operation, params=None, *, tags=None):
```

Comment:
Could you move `tags` to connection from query? For consistency with JDBC driver and CLI.

Reply:
@ebyhr thanks for the review. I can certainly add it to connection as well. Since the change to execute is non-destructive, would you mind making this a superset of the other API's and leaving it in execute as well? We have use-cases where this would make sense on a per-query basis.

Reply:
Can you share the details of your use-case? Since it's "client"-tags, per-query basis might be confusing to users. (=client won't change per query)

Reply:
Ok, I will make the change. The use cases I had in mind can also be handled if the tags are passed in the connection.